standard/expressions.md

```diff
@@ -1187,7 +1187,7 @@ Even though overload resolution of a dynamically bound operation takes place at
 In these cases a limited compile-time check is performed on each member in the known set of function members, to see if it can be known for certain never to be invoked at run-time. For each function member `F` a modified parameter and argument list are constructed:
 
 - First, if `F` is a generic method and type arguments were provided, then those are substituted for the type parameters in the parameter list. However, if type arguments were not provided, no such substitution happens.
-- Then, any parameter whose type is open (i.e., contains a type parameter; see [§8.4.3](types.md#843-open-and-closed-types)) is elided, along with its corresponding parameter(s).
+- Then, any parameter whose type is open (i.e., contains a type parameter; see [§8.4.3](types.md#843-open-and-closed-types)) is elided, along with its corresponding argument.
```

Comment:
This one is worth discussing. I don't think there's such a thing as a "corresponding parameter for a parameter" - the correspondence is between arguments and parameters (unless I've missed something).

Arguably this should be "any argument whose corresponding parameter's type is open is elided, along with that corresponding parameter". I'd be happy to change it to that if folks would prefer it.

Reply:
I don’t think you’ve missed something, but it should be *arguments* – the parameter in question could be a `params` (e.g. `params T[] args`) and so have multiple corresponding arguments.

This would also I think argue against your optional alternate wording.

```suggestion
- Then, any parameter whose type is open (i.e., contains a type parameter; see [§8.4.3](types.md#843-open-and-closed-types)) is elided, along with its corresponding arguments.
```

Reply:
Yup, I was wondering about that. I've just found that we use "corresponding argument" in various places, which I didn't think we did. Will commit your suggestion now.